README.md

```diff
@@ -114,6 +114,14 @@ Interface | `ng g interface my-new-interface`
 Enum      | `ng g enum my-new-enum`
 Module    | `ng g module my-module`
 
+angular-cli will add reference to `components`, `directives` and `pipes` automatically in the `app.module.ts`. If you need to add this references to another custom module, follow this steps:
+
+1. `ng g module newModule` to create a new module
+2.  you should change directory with `cd newModule` into the `newModule` folder
```

Comment:
I change it and commit it. There's anything else I need to do? I've seen that my commit update the PR but I don't know whether I need to to something else. Sorry for these silly questions, I not used to do PR, reviews, etc.